supplementary/alica_ros_turtlesim/ros1/CMakeLists.txt

```diff
@@ -91,10 +93,10 @@ install(TARGETS ${PROJECT_NAME}
         RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
 )
 
-install(DIRECTORY include/${PROJECT_NAME}/
+install(DIRECTORY include/${PROJECT_NAME}/ ../include
         DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
 )
 
-install(DIRECTORY launch etc
+install(DIRECTORY launch etc ../etc
```

Comment:
seems wrong.  We should only need plans from one place

Reply:
The local etc folder contains the ros specific behaviours WaitForTrigger and WaitForMsg, while the etc folder of the parent directory contains all shared plans / behaviours like SurroundLeader.

Reply:
Shouldn't this be installed by the CMakeLists of the parent folder though?

Reply:
This doesnt really work with our current design of having a non package parent folder. We could have an alica_turtlesim_base folder at the same level as ros1 / ros2 with the etc folder, then add it as a subdir to ros1 / ros2 and depending on if we find ament_cmake, install the etc folder of alica_turtlesim_base to install/share/alica_ros2_turtlesim to merge the etc folder of ros2 and alica_turtlesim_base.